config/manager/manager.yaml

```diff
@@ -16,12 +16,22 @@ spec:
   selector:
     matchLabels:
       control-plane: controller-manager
-  replicas: 1
+  replicas: 2
   template:
     metadata:
       labels:
         control-plane: controller-manager
     spec:
+      affinity:
+          podAntiAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Thought: Would it be better to go with `preferredDuringSchedulingIgnoredDuringExecution` so that 2 pods would still be created even if the operator was being used on a single node cluster. 

>There are currently two types of node affinity, called requiredDuringSchedulingIgnoredDuringExecution and preferredDuringSchedulingIgnoredDuringExecution. You can think of them as “hard” and “soft” respectively, in the sense that the former specifies rules that must be met for a pod to be scheduled onto a node (just like nodeSelector but using a more expressive syntax), while the latter specifies preferences that the scheduler will try to enforce but will not guarantee.

https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity

Reply:
I think that's a good point. I was considering this myself and you've pushed me over the edge